Day12_Basic_ofApex.cls

```diff
@@ -0,0 +1,119 @@
+public with sharing class Day12_Basic_ofApex {
+    public Day12_Basic_ofApex(Integer choice) {
+        if(choice==1){
+            implementExpression();
+        }
+        else if(choice==2){
+            implementExpressionOperator();
+
+        }
+        else if(choice==3){
+            implementSafeNavigation();
+        }
+        else if(choice==4){
+            implementComments();
+        }
+        else if(choice==5){
+            implementRulesofConversion();
+        }
+        else{
+            system.debug('you have choosen wrong try again');
+    }   }
+    public static void  implementExpression(){
+        integer first=1+1;
+        system.debug(first);
+        //for list map integer set sobject
+        Account acc= new Account();
+        list<Account> l=new List<Account>();
+        set<String> s1=new Set<String>{};
+        Map<String, Integer> m1=new Map<String, Integer>();
+        integer i = [SELECT COUNT() FROM Contact WHERE LastName ='Weissman'];
```

Comment:
Use logical name for the variable 'i'. Always use capital letters for variable type. Use Integer instead of integer.